librz/core/cmd_seek.c

```diff
@@ -171,8 +171,8 @@ static void cmd_seek_opcode(RzCore *core, const char *input, bool silent) {
 		n = 1;
 	}
 	int val = (n < 0)
-		? cmd_seek_opcode_backward(core, -n, silent)
-		: cmd_seek_opcode_forward(core, n, silent);
+		? rz_core_seek_opcode_backward(core, -n, silent)
```

Comment:
I suggest to extract a `rz_core_seek_opcode` that calls the right function based on whether n > 0 or < 0.

Reply:
Done.